Ivanchuk_Pavel/03.Git.Hosting/autopush.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+#List of remote repositories
+remote_repositories=("github-origin" "gitlab-origin")
```

Comment:
Instead of declare this array you can use `$(git remote)` to fetch all remote links